vegetason/Silicon-Farm
# Finish setting up the app

### PR Description

This PR creates adds necessary configuration of the app and test the app.

### How can this been tested?
- Clone repo
- Run `npm install`, then `npm run dev` to check if the app is running successfully
